src/schema/src/transaction_pool.rs

```diff
@@ -243,3 +243,90 @@ where
 lazy_static! {
     pub static ref POOL: Pool = Pool::new();
 }
+
+
+#[cfg(test)]
+mod tests_transaction_pool {
+
+    use super::*;
+    use std::collections::HashMap;
+    use utils::crypto::keypair::{CryptoKeypair, Keypair, KeypairType};
+    use utils::serializer::{deserialize, serialize, Deserialize, Serialize};
+    use std::time::SystemTime;
+    pub use sdk::signed_transaction::SignedTransaction;
+
+    #[derive(Debug, Clone, PartialEq, Serialize, Deserialize, BinaryValue, ObjectHash)]
+    #[binary_value(codec = "bincode")]
+    pub struct CryptoTransaction {
+        pub from: std::string::String,
+        pub fxn_call: std::string::String,
+        pub payload: std::vec::Vec<DataTypes>,
+    }
+    #[derive(Debug, Clone, PartialEq, Serialize, Deserialize, BinaryValue, ObjectHash)]
+    #[binary_value(codec = "bincode")]
+    pub enum DataTypes {
+        BoolVal(bool),
+        IntVal(i32),
+        HashVal(Hash),
+        StringVal(String),
+        VecHashVal(Vec<Hash>),
+        VecStringVal(Vec<String>),
+    }
+
+    #[test]
+    pub fn test_transaction_pool() {
+        let temp_pool: Pool = Pool::new();
+        const txn_fxn_arr: [&'static str; 5] = ["transfer_sc", "set_hash", "add_doc", "transfer_for_review", "review_docs"];
+        let mut stxn_arr = vec![];
+        for fxn in txn_fxn_arr.iter()
```

Comment:
Is it possible to have separate unit test cases for "insert operation" & "sync pool operation"?